pennylane/data/dataset.py

```diff
@@ -186,9 +188,19 @@ def read(self, filepath, lazy=False):
         if attribute == "full":
             data = self._read_file(filepath)
             for attr, value in data.items():
-                setattr(self, f"{attr}", None if lazy else value)
+                if lazy:
```

Comment:
Is this `lazy` independent of the `lazy` argument in the `load`? Otherwise, it would be true by default for the standard datasets.

Reply:
Just saw the next PR, to be sure, are we removing `lazy` from here?

Reply:
we are ultimately removing lazy from here, yes. I forgot about lazy in`load`, it's totally unused! I can add it in the next PR